mapping_test.go

```diff
@@ -0,0 +1,140 @@
+package gorp
+
+import (
+	"reflect"
+	"sync"
+	"testing"
+	"time"
+)
+
+type testUser struct {
+	ID             uint64    `db:"id"`
+	Username       string    `db:"user_name"`
+	HashedPassword []byte    `db:"hashed_password"`
+	EMail          string    `db:"email"`
+	CreatedAt      time.Time `db:"created_at"`
+	UpdatedAt      time.Time `db:"updated_at"`
+}
+
+type testCoolUser struct {
+	testUser
+	IsCool      bool     `db:"is_cool"`
+	BestFriends []string `db:"best_friends"`
+}
+
+func BenchmarkCcolumnToFieldIndex(b *testing.B) {
```

Comment:
Minor spelling mistake here: `Ccolumn` should be `Column`

Reply:
Fixed in b9b8642